htmlbars-version/app/components/focusing-outlet.js

```diff
@@ -0,0 +1,14 @@
+import Ember from "ember";
+import FocusingOutletMixin from 'ember-a11y/mixins/focusing';
+
+const { Component } = Ember;
+
+let FocusingOutlet = Component.extend(FocusingOutletMixin, {
+  positionalParams: ['inputOutletName'] // needed for Ember 1.13.[0-5] and 2.0.0-beta.[1-3] support
```

Comment:
TBH, I think we can probably just say "screw it" for 1.13.0 - 1.13.5. There is absolutely no reason for someone to use 1.13.5 instead of latest 1.13.x.